server/src/ClientToDBAPI/InvestigationInfo/mainRoute.ts

```diff
@@ -297,4 +296,27 @@ investigationInfo.get('/subStatuses', (request: Request, response: Response) =>
         });
 });
 
+investigationInfo.get('/statuses', (request: Request, response: Response) => {
+    graphqlRequest(GET_STATUSES, response.locals, {})
```

Comment:
Why sending an empty object? I think you can just not send it at all

Reply:
fixed